.ci/azure-pipelines/env.yml

```diff
@@ -58,6 +58,13 @@ jobs:
         # nvidia-cuda docker image has not been released for 20.04 yet
```

Comment:
Is not correct anymore - https://hub.docker.com/layers/nvidia/cuda/11.1-devel-ubuntu20.04/images/sha256-c950a0040c0e4c1503834f2ad2ec08d181af72095673c7500dd24a1aacb84893?context=explore